snakemake-mode.el

```diff
@@ -325,6 +328,56 @@ label."
     (nreverse index)))
 
 
+;;; Embedded language syntax-highlighting
+
+(defun snakemake-mode-setup-mmm ()
+  "Set up MMM mode to highlight embedded R code.
+
+You must have the R-strings either within a R(''' ''') function call or a code
+block delimited with '''#r and ''' (triple double-quotes also accepted).
+
+For automatic highlighting of embedded regions, you need to set
+`mmm-global-mode` to 'maybe in your setup file."
+
+  (require 'mmm-mode)
```

Comment:
Having this `require` here would result in an error if `mmm-mode`
isn't installed, and the next `require` call would not be reached.